src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -298,7 +296,11 @@ public static KNNQueryBuilder fromXContent(XContentParser parser) throws IOExcep
             }
         }
 
-        validateSingleQueryType(k, maxDistance, minScore);
+        VectorQueryType vectorQueryType = validateSingleQueryType(k, maxDistance, minScore);
```

Comment:
rename the function of getQueryType.

Reply:
@navneet1v This function meanwhile is used in some other places for validation, so the name of `getQueryType` might isn't suitable I think. How about `handleQueryType`?

https://github.com/opensearch-project/k-NN/blob/9cdbcb2fd393ae77ec30603c929a772c10f3ea3a/src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java#L104

https://github.com/opensearch-project/k-NN/blob/9cdbcb2fd393ae77ec30603c929a772c10f3ea3a/src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java#L121

https://github.com/opensearch-project/k-NN/blob/9cdbcb2fd393ae77ec30603c929a772c10f3ea3a/src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java#L135